src/prices/PricesController.ts

```diff
@@ -12,7 +13,7 @@ export class PriceController {
   @ContentType('json')
   @Get('/')
   async listPrices(
-    @QueryParams('chain') chain?: string,
+    @QueryParams('chain') chain?: Network,
```

Comment:
super nit: Should we rename the parameters to `network` or make the `Network` interface more explicit (like `ChainNetwork`)?